### ISSUE TITLE
Cannot download attachments from chat

### ISSUE THREAD

**Issue by tetianakh** [keybase/client#4740]
First of all, thanks a lot for the awesome project! I'm trying it out and it's very nice. I'm especially excited about the command line chat feature. However, I can't seem to figure out how to use attachments.

Whenever I try to download an attachment sent to me in chat, I get  "ERROR unencrypted attachments not supported" -- even if attachment is encrypted.

The sender can download the same attachment without a problem.

Any ideas how to fix this?

**Comment by maxtaco** [comment]
cc: @patrickxb

**Comment by patrickxb** [comment]
Chat has not been released or announced and everything should be considered pre-alpha.

We aren't supporting old (unencrypted) attachments.  We didn't know anyone in the public was using it.  The sender has an old client and can thus download them.

**Comment by rkujawa** [comment]
I am the sender and I am running keybase version 1.0.17-20161025160021+77565f4. This is considered "old"? ;)

Btw. let me clarify. I have uploaded by using:
```
$ keybase chat upload me,her ~/a.file
```

When we have noticed this isn't working, I have also tried first encrypting the file via `keybase encrypt`, then uploading it in the same way. But when @tetianakh tried to download it, it resulted in the same error. But I guess the file should be transparently encrypted when doing `chat upload` ?

**Comment by patrickxb** [comment]
Yes, that's "old".  This is under active development.  And yes, newer clients transparently encrypt.

Just so you know, we have deleted the chat database (and all attachments) several times during development and you should be prepared that it will happen again before it is released.

**Comment by maxtaco** [comment]
BTW, @rkujawa and @tetianakh thanks for your enthusiasm and helping us to debug!

**Comment by rkujawa** [comment]
Ok, thanks for the information. This feature is awesome though, keep up the good work!

**Comment by patrickxb** [comment]
Thanks!  We're excited to release it.
